main/certs/root_cert_auth.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Probably want to remove these credentials?

Reply:
It is the [Amazon RootCA1](https://www.amazontrust.com/repository/AmazonRootCA1.pem) in PEM format, I think it is okay to have it _in_ the repo, since it is a pre-requisite to establishing the TLS connection successfully.